hack/gen-disk-skus-map.go

```diff
@@ -0,0 +1,333 @@
+/*
```

Comment:
add build test for this go files? it's better put into pkg folder which has unit test coverage

Reply:
@andyzhangx Sure. Do you have an specific thing in mind? Or may be an example?

Also, since this is not in the hot path, please allow me to address this as a chore in a separate PR. I am evaluating to see weather I should spend time importing this information dynamically via the cloud provider, so that might be a better investment of our time.